### DISCUSSION TITLE
Welcome to fwaopt Discussions!

### DISCUSSION THREAD

**Discussion by cil-lab** [cil-lab/fwaopt#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about fireworks algorithm
  * Share ideas related to fireworks algorithm.

**Comment by 1669716869** [comment]
是否有关于烟花算法在可见光室内定位方向的研究，可否提供研究代码以供学习
